osu.Game/Beatmaps/Legacy/LegacyStoryLayer.cs

```diff
@@ -3,11 +3,12 @@
 
 namespace osu.Game.Beatmaps.Legacy
 {
-    internal enum LegacyStoryLayer
+    public enum LegacyStoryLayer
```

Comment:
legacy classes should not be public. use strings for now in places that are non-legacy (ie. `Storyboard.cs`)